plaintext_mnist_1/mnist_1.py

```diff
@@ -12,29 +16,103 @@
 MODEL_PATH = 'mnist_ffnn_model.pth'
 RANDOM_SEED = 42 # for reproducibility
```

Comment:
Maybe we could call it something else than ‘RANDOM_SEED’ to avoid potential confusion? Maybe ‘RNG_SEED’?